### PR TITLE
[Nuclio] Add container image to function status

### PR DESCRIPTION
**Pull request by TomerShor**
Add the built and pushed container image name, that is used to run the nuclio function container, to the function status.

A followup to https://github.com/nuclio/nuclio/pull/2769 released in https://github.com/nuclio/nuclio/releases/tag/1.11.7 .

Fixes https://jira.iguazeng.com/browse/IG-21462

### CODE REVIEW

**Review by liranbg — APPROVED** [review]

File: tests/system/runtimes/test_nuclio.py [hunk 1]
```diff
@@ -91,6 +91,43 @@ def test_workaround_for_nuclio_bug(self):
...
+        self._logger.debug("Creating nuclio function")
+        function = mlrun.code_to_function(
+            name="test-function",
+            kind="serving",
+            project=self.project_name,
+            filename=code_path,
+            image="mlrun/mlrun",
+        )
+
+        graph = function.set_topology("flow", engine="sync")
```
Comment (AlonMaor14) [inline tests/system/runtimes/test_nuclio.py:106]: Is this related to the test? looks like it can be removed

**Review by AlonMaor14 — CHANGES_REQUESTED** [review]
Nice job! minor comment and please enrich the container image in `mlrun.runtimes.function.RemoteRuntime._get_state`

File: tests/system/runtimes/test_nuclio.py (hunk 1 above)
Comment (TomerShor) [reply tests/system/runtimes/test_nuclio.py:106]: Since I'm deploying a serving function in the test, it must have a graph topology.

**Review by liranbg — APPROVED** [review]
Well done adding comments where needed/missing

**Review by Tankilevitch — APPROVED** [review]
